configs/odroid/common/scripts/format_sd.sh

```diff
@@ -1,17 +1,10 @@
 #!/bin/bash
-
```

Comment:
Sorry, I've moved some check on top of the script (for consistency with other scripts), it make the diff harder to read than it shoulb be.